extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -166,6 +171,20 @@ export class AmpSlideScroll extends BaseSlides {
     return this.noOfSlides_ > 2;
   }
 
+  /** @override */
+  mutatedAttributesCallback(mutations) {
+    mutations.forEach(mutation => {
+      const newValue = mutation.newValue;
+      switch (mutation.name) {
+        case 'slide':
+          const slide = parseInt(newValue, 10);
+          user().assert(isFinite(slide), 'Invalid [slide] value: %s', newValue);
+          this.showSlide_(slide);
```

Comment:
`mutatedAttributesCallback` is not a direct result of user action; it's called when a bound attribute has its value changed during a Bind digest.

When to fire events/actions is a design choice in Bind to avoid cycles. More discussion [here](https://docs.google.com/document/d/1oQeCELbc_jjwH7MiP8kohYlHcP_XSXlPwhqPQ-KgVhI/edit?disco=AAAAA3LaybQ).